projects/lib/src/board/twokingseachboard.cpp

```diff
@@ -0,0 +1,206 @@
+/*
+    This file is part of Cute Chess.
+
+    Cute Chess is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Cute Chess is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Cute Chess.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "twokingseachboard.h"
+#include "westernzobrist.h"
+
+namespace Chess {
+
+TwoKingsEachBoard::TwoKingsEachBoard()
+	: WesternBoard(new WesternZobrist()),
+	  m_symmetrical(isSymmetrical()),
```

Comment:
I think you forgot the `m_symmetrical` initialization here because it's also in `vInitialize()`. And of course virtual member functions shouldn't be called from the constructor anyway.

I think it's still a good idea to give `m_symmetrical` the default value in the constructor (to avoid undefined state/behavior) but without calling `isSymmetrical()`.